static/analysis.go

```diff
@@ -83,53 +87,97 @@ func analyzeIdentityCall(c *ssa.Call) configkit.Identity {
 	return ident
 }
 
-// analyzeHandler analyzes a type that implements one of the Dogma handler
-// interfaces to deduce its handler configuration.
-func analyzeHandler(
+// addHandlerFromArguments adds a handler to the handler set from the given
+// arguments. It assumes that the handler is always the first argument.
+//
+// If the first argument is not a pointer to ssa.MakeInterface instruction, this
+// function has no affect.
```

Comment:
```suggestion
// addHandlerFromArguments analyzes args to deduce the configuration of a
// handler of type ht. It assumes that the handler is the first argument.
//
// If the first argument is not a pointer to ssa.MakeInterface instruction, this
// function has no effect; otherwise the handler is added to hs.
```